scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -307,26 +306,37 @@ For this example we will focus on identifying DE genes in these `Tumor_Myoblast`
 
 ## Differential Expression Analysis
 
+Now we are ready to start preparing for our DE analysis, where we will compare the gene expression of all cells annotated as `Tumor_Myoblasts` in 3 ARMS samples to 3 ERMS samples.
+
+Throughout the notebook we have been working with an integrated dataset that contains corrected gene expression data and a corrected UMAP.
+However, we do not want to use corrected gene expression values for differential expression, we want to use the original raw counts that are present in the `counts` assay of the `SingleCellExperiment` object. 
+It is advised to only use the corrected values for any analyses being performed at the cell level, e.g. dimensionality reduction.
+In contrast, it is not advised to use corrected values for any analyses that are gene-based, such as differential expression or marker gene detection, because within-batch and between-batch gene expression differences are no longer preserved.
+See the [OSCA chapter on Using the corrected values](http://bioconductor.org/books/3.15/OSCA.multisample/using-corrected-values.html#using-corrected-values) for more insight.
+
+For the remainder of the DE analysis we will be using data found in the integrated object that has been uncorrected. 
+As a reminder, the uncorrected gene expression data, in the `counts` and `logcounts` assays, correspond to data that has been merged (the first step we walked through prior to integration) but has not been corrected.
```

Comment:
The phrasing feels a but weird to say "the uncorrected data are the data that has not been corrected".. I'm not necessarily sure I've suggested better phrasing though!

```suggestion
As a reminder, the uncorrected gene expression data, found in the `counts` and `logcounts` assays, correspond to data that has been merged (the first step we walked through prior to integration) into the same SCE but not yet integrated.
```